flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveLookupFunction.java

```diff
@@ -134,9 +188,48 @@ public void eval(Object... values) {
 		}
 	}
 
-	@VisibleForTesting
-	public Duration getCacheTTL() {
-		return cacheTTL;
+	private HiveTableInputFormat getHiveTableInputFormat() {
```

Comment:
Can we add a `copy(Partitions)` method to `HiveTableInputFormat`? Then we don't need move so many logicals.